aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -156,6 +157,48 @@ storage:
             --net=host \
             --dns=host \
             --exec=/bootkube -- start --asset-dir=/assets "$@"
+    - path: /opt/etcd-rejoin
+      filesystem: root
+      mode: 0555
+      contents:
+        inline: |
+          #!/bin/bash
+          set -eou pipefail
+          # Script to rejoin as fresh node to an existing etcd cluster when the node cannot join.
+          # The script should be added to /etc/systemd/system/etcd-member.service.d/40-etcd-cluster.conf
+          # as ExecStopPost=-/opt/etcd-rejoin because it should run when the etcd service failed and because it
+          # makes use of the environment variables set there for etcd-member.service.
+          # It only runs when etcd failed to join the cluster (e.g., after repovisioning the node or after
+          # the node was down for too long and got kicked out).
+          # Skip if node was not fully provisioned yet.
+          if [ ! -d "/etc/ssl/etcd/" ]; then exit 0; fi
+          # Service just got stopped
+          if [ "$EXIT_CODE" = "killed" ]; then exit 0; fi
+          export ETCDCTL_API=3
+          urls=$(echo "$ETCD_INITIAL_CLUSTER" | tr "," "\n" | cut -d "=" -f 2 | tr "\n" "," | head -c -1)
+          # Double $$ for terraform
+          endpoints="$${urls//2380/2379}"
```

Comment:
Don't know what `//2380/2379` is. It seems like ports, right? Is it to try one port and if not the other one? Just curious, not that is seems wrong or anything :D

Reply:
This is bash string replacement. The URLs need to have the 2379 port.

Reply:
:ok_hand: